element_array_ephys/ephys_acute.py

```diff
@@ -84,6 +99,18 @@ def get_session_directory(session_key: dict) -> str:
     return _linking_module.get_session_directory(session_key)
 
 
+def get_processed_root_data_dir() -> str:
+    """
+    get_processed_root_data_dir() -> str:
+        Retrieves the root directory for all processed data to be found from or written to
+        :return: a string for full path to the root directory for processed data
+    """
+
+    if hasattr(_linking_module, 'get_processed_root_data_dir'):
+        return _linking_module.get_processed_root_data_dir()
+    else:
+        return get_ephys_root_data_dir()[0]
```

Comment:
```suggestion
        return get_ephys_root_data_dir()[0] if isinstance(get_ephys_root_data_dir(), list) else get_ephys_root_data_dir()
```

Reply:
`get_ephys_root_data_dir` should always return as a list

Reply:
The [import from linking_module](https://github.com/ttngu207/element-array-ephys/blob/no-curation/element_array_ephys/ephys_no_curation.py#L83) enforces list type

Reply:
Plus `is_instance(thing, list)` is not a robust way to check if thing is a sequence, it could be a tuple for example.